numba_cuda/numba/cuda/extending.py

```diff
@@ -3,5 +3,56 @@
 """
 
 from numba.core.extending import intrinsic as _intrinsic
+from numba.cuda.models import register_model  # noqa: F401
+from numba.cuda import models  # noqa: F401
 
 intrinsic = _intrinsic(target="cuda")
+
+
+def make_attribute_wrapper(typeclass, struct_attr, python_attr):
```

Comment:
This could do with a comment mentioning that it's vendored from `numba.core.extending` with a change to consider the CUDA data model manager, for the benefit of future readers of the code outside of the context of the PR.